src/OrchardCore.Modules/OrchardCore.Shortcodes/Providers/LocaleShortcodeProvider.cs

```diff
@@ -9,19 +9,19 @@ public class LocaleShortcodeProvider : IShortcodeProvider
     {
         public const string ShortCodeIdentifier = "locale";
 
-        private static readonly ValueTask<string> Null = new ValueTask<string>((string)null);
+        private static readonly ValueTask<string> Empty = new ValueTask<string>(String.Empty);
 
         public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, string content, Context context)
         {
             if (identifier != ShortCodeIdentifier)
             {
-                return Null;
+                return Empty;
```

Comment:
This should return Null as before

Reply:
Dean if you look before my last commit is was `Null` the issue comes from `null` value

Reply:
I checked shortcodes. This is wrong.

Please apply the suggestions, return Null initially, and Empty if it has no value.

Reply:
Let me try your suggestions